tests/ha/test_multi_server_pod_failure.py

```diff
@@ -1376,3 +1377,506 @@ def test_part_mpu_till_kserver_pods_fail(self):
+
+        LOGGER.info("Step 6: Check services status on remaining pods %s", server_data_pod_list)
+        resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=server_data_pod_list,
+                                                           fail=False)
+        LOGGER.debug("Response: %s", resp)
+        assert_utils.assert_true(resp[0], resp)
+        LOGGER.info("Step 6: Services of remaining pods are in online state")
+        event.clear()
+
+        LOGGER.info("Step 7: Verifying response of background process")
+        thread.join()
+        while True:
+            resp = output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+            if isinstance(resp, bool):
+                break
+
+        if resp is None:
+            assert_utils.assert_true(False, "Background process of chunk upload failed")
+        LOGGER.info("Step 7: Successfully verified response of background process")
+
+        if not resp:
+            LOGGER.info("Step 8: Chunk upload failed in between, trying chunk upload again")
+            self.ha_obj.create_bucket_chunk_upload(s3_data=self.s3_clean,
+                                                   bucket_name=self.bucket_name,
+                                                   file_size=file_size,
+                                                   chunk_obj_path=chunk_obj_path,
+                                                   output=output,
+                                                   bkt_op=False)
+
+            while True:
+                resp = output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+                if isinstance(resp, bool):
+                    break
+
+            if not resp or resp is None:
+                assert_utils.assert_true(False, "Retried chunk upload failed")
+            LOGGER.info("Step 8: Retried chunk upload completed successfully")
+
+        LOGGER.info("Calculating checksum of file %s", chunk_obj_path)
+        upload_checksum = self.ha_obj.cal_compare_checksum(file_list=[chunk_obj_path],
+                                                           compare=False)[0]
+
+        LOGGER.info("Step 9: Download object and verify checksum")
+        resp = s3_test_obj.object_download(self.bucket_name, self.object_name, download_path)
+        LOGGER.info("Download object response: %s", resp)
+        assert_utils.assert_true(resp[0], resp[1])
+        download_checksum = self.ha_obj.cal_compare_checksum(file_list=[download_path],
+                                                             compare=False)[0]
+        assert_utils.assert_equal(upload_checksum, download_checksum,
+                                  f"Failed to match checksum: {upload_checksum},"
+                                  f" {download_checksum}")
+        LOGGER.info("Step 9: Successfully downloaded object and verified checksum")
+
+        LOGGER.info("Step 10: Create new IAM user and start IOs, buckets and upload objects")
```

Comment:
We don't need this step. As we are already checking the same in specific tests.

Reply:
This is required.